nahand/package.json

```diff
@@ -2,9 +2,20 @@
   "name": "test",
   "version": "0.0.0",
   "main": "main.js",
+  "scripts": {
+    "start": "electron .",
+    "prepublish": "bower install && npm run build",
```

Comment:
Oh! Good question, I forgot to explain that point. This script is automatically executed by npm after running `npm install`. There's also `postinstall` that do the same thing. The main difference is that `postinstall` script is executed in two scenarios where this is inappropriate : when the project is installed globally or when installed as a dependency. In that case, it will typically fail because of the fact that dev dependencies are not installed in none of those contexts. Which makes sense as  `postinstall` is really about having a script being ran after a consumer of the lib/bin have installed it. 

The full list of life cycle scripts is available in the doc for package.json.